aptos-move/framework/aptos-stdlib/sources/simple_map.spec.move

```diff
@@ -56,12 +62,31 @@ spec aptos_std::simple_map {
         pragma verify=false;
     }
 
-    spec to_vec_pair {
-        pragma verify=false;
+    spec to_vec_pair<Key: store, Value: store>(map: SimpleMap<Key, Value>): (vector<Key>, vector<Value>) {
+        pragma intrinsic;
+        pragma opaque;
+        ensures [abstract] result_1 == spec_to_vec_keys(map);
+        ensures [abstract] result_2 == spec_to_vec_values(map);
```

Comment:
Do we necessarily need this (and hence the two uninterpreted functions above)?
Or the following two `ensures` are enough?